### ISSUE TITLE
Geocaching not working for groups in admin panel

### ISSUE THREAD

**Issue by sebgrebe** [rtcharity/eahub.org#440]
*Problem*:
When adding the location of a group in admin panel through the "City or Town" and "Country" fields, the group does not show up on the map. It does work when putting lat/lon in though.
Test case: London, UK

**Comment by taymonbeal** [comment]
Yeah, the admin panel has you manually specify the map location. I think it's safe for this to wait until Plato? Admins can edit groups through the regular UI until then.

**Comment by sebgrebe** [comment]
> I think it's safe for this to wait until Plato

Yes, definitely.

**Comment by taymonbeal** [comment]
Probably this will be fixed as part of the geocoding overhaul when that happens.

**Comment by taymonbeal** [comment]
Do we need a temporary patch for this until we fix geocoding more permanently, or can it wait?

**Comment by michaltrzesimiech** [comment]
This surely can wait
